src/components/footer.component.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+const StyledFooter = styled.footer`
+  background-color: #282c34;
+  height: 40px;
+  color: white;
+  font-size: 20px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+  //TODO refactor to link style
+  &:visited {
+    color: white;
+  }
+  & a {
+    color: white;
+    text-decoration: none;
+  }
```

Comment:
This should be declared in the div that's inside this element

Reply:
Oki :) Moved all links styled to another component links in common :)